{
    "type": "bundle",
    "id": "bundle--b2ed0027-2892-495b-9f50-2eeb2980f1b2",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--0192fbe8-96de-47e1-9555-52d86616bb26",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-03-19T00:20:51.653921Z",
            "modified": "2025-03-19T00:20:51.653921Z",
            "name": "CVE-2024-23943",
            "description": "An unauthenticated remote attacker can gain access to the cloud API due to a lack of authentication for a critical function in the affected devices. Availability is not affected.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-23943"
                }
            ]
        }
    ]
}